packages/big-design/src/components/GucciBox/GucciBox.tsx

```diff
@@ -0,0 +1,22 @@
+import { VariantProps } from '@stitches/react';
+import React, { forwardRef, HTMLAttributes, memo } from 'react';
+
+import { StyledGucciBox } from './styled';
+
+type StyledVariants = VariantProps<typeof StyledGucciBox>;
+
+export interface BoxProps extends HTMLAttributes<HTMLDivElement>, StyledVariants {
+  as?: keyof JSX.IntrinsicElements | React.ComponentType<any>;
+}
+
+interface PrivateProps {
+  forwardedRef: React.Ref<HTMLDivElement>;
+}
+
+const RawBox: React.FC<BoxProps & PrivateProps> = (props) => <StyledGucciBox ref={props.forwardedRef} {...props} />;
```

Comment:
We need to switch this up a bit because of passed props
```suggestion
const RawBox: React.FC<BoxProps & PrivateProps> = ({ forwardedRef, ...props }) => (
  <StyledGucciBox ref={forwardedRef} {...props} />
);
```